pkg/rootless/rootless_linux.go

```diff
@@ -365,7 +365,7 @@ func GetConfiguredMappings() ([]idtools.IDMap, []idtools.IDMap, error) {
 	}
 	mappings, err := idtools.NewIDMappings(username, username)
 	if err != nil {
-		logrus.Warnf("cannot find mappings for user %s: %v", username, err)
+		logrus.Errorf("cannot find mappings for user %s: %v", username, err)
 	} else {
```

Comment:
Would it be a problem for people using a single ID mapped to get always the error?

Reply:
That's what was happening before, so I kept it. We can swap it back if folks complain, I think.